app/controllers/pages_controller.rb

```diff
@@ -28,6 +28,12 @@ def bounty
     set_surrogate_key_header "bounty_page"
   end
 
+  def code_of_conduct
```

Comment:
This and other functions on this page allow Pages to override the hardcoded template